dev-python/jsonschema-spec/jsonschema-spec-0.2.0.ebuild

```diff
@@ -29,6 +29,12 @@ RDEPEND="
 	>=dev-python/jsonschema-specifications-2023.3.4[${PYTHON_USEDEP}]
 "
 
+BDEPEND="${RDEPEND}
```

Comment:
This part is unnecessary. BDEPEND aren't required for non-test install, and distutils_enable_tests takes care of copying them into `test?` block.